README.rst

```diff
@@ -104,11 +104,41 @@ on the command line:
 | | ``--json``      | Deep tokenization. Output token objects in JSON   |
 |                   | format, one per line.                             |
 +-------------------+---------------------------------------------------+
-| | ``--normalize`` | Normalize punctuation, causing e.g. quotes to be  |
+
+Other options can be specified on the command line:
+
++-------------------+---------------------------------------------------+
+| | ``--n``         | Normalize punctuation, causing e.g. quotes to be  |
 |                   | output in Icelandic form and hyphens to be        |
 |                   | regularized. This option is only applicable to    |
 |                   | shallow tokenization.                             |
 +-------------------+---------------------------------------------------+
+| | ``--s``         | Input contains one sentence per line.             |
++-------------------+---------------------------------------------------+
+| | ``--m``         | Degree signal in tokens denoting temperature      |
+|                   | normalized (200° C -> 200 °C)                     |
++-------------------+---------------------------------------------------+
+| | ``--a``         | Additional annotation, usually handled by         |
```

Comment:
What is this? Must confess I don't know!

Reply:
It makes Tokenizer use parse_phrases_2(), combining more complex dates and currency etc., which GreynirPackage would usually handle.